app/views/layouts/application.html.slim

```diff
@@ -0,0 +1,12 @@
+doctype html
+html
+  head
+    title SagnCustomersWeb
```

Comment:
Better to move it to separated partial `views/shared_head.html.slim`, to avoid code duplication.